guides/common/modules/proc_performing-post-upgrade-tasks.adoc

```diff
@@ -34,3 +34,12 @@ ifdef::satellite,orcharhino[]
 This migration takes time, so a pre-migration runs automatically after the upgrade to {ProjectVersion} to reduce future upgrade downtime.
 While the pre-migration is running, {ProjectServer} is fully functional but uses more hardware resources.
 endif::[]
+
+. Optional: Delete the `/var/lib/pgsql/data-old/` directory by entering the following command:
++
+[options="nowrap" subs="attributes"]
+----
+# rm /var/lib/pgsql/data-old/
+----
++
+{ProductName} creates this directory for backup purposes during the upgrade. If the upgrade has finished successfully, the directory is no longer needed.
```

Comment:
```suggestion
{Project} creates this directory for backup purposes during the upgrade.
If the upgrade has finished successfully, the directory is no longer needed.
```
* s/{ProductName}/{Project}/ because `{Project}` is the attribute used to refer to Foreman in general. This should also fix the foreman-el build failure.
* We (do our best to) follow one sentence per line.